src/util/timezones.cpp

```diff
@@ -16,6 +16,8 @@
 #include <string>
 #include <unordered_map>
 
+#include <time.h>
```

Comment:
it is ok both `localtime` versions are declared `time.h` in http://pubs.opengroup.org/onlinepubs/009695399/functions/localtime.html
somewhere the file was already included in non win32 builds